h5p-bildetema/src/components/Select/Select.module.scss

```diff
@@ -36,26 +36,29 @@
   border-radius: 0.5rem;
   padding: 0.25rem;
   position: absolute;
-  margin-top: 3.5rem;
-  z-index: $fav-language-selector-z-index;
+  margin-top: 3rem;
+  z-index: 11;
```

Comment:
The used z-indexes should be stored as variables in the same location, so that it is easy to have an overview of them and to set z-index relative to each other. Do we need to change this? If so, make a new variable or change the existing

Reply:
Ahh yes, i forgot 😅